File: C programming/Selection1.c
#include<stdio.h>
// Accept number from user and check whether that number is even or odd
int main()
{
    int No = 0;
    int Ans = 0;

    printf("Enter number : \n");
    scanf("%d",&No);

    Ans = No % 2;

    if(Ans == 0)    // Jar
    {
        printf("Number is even\n");
    }
    else    // Nasel tar
    {
        printf("Number is odd\n");
    }

    return 0;
}

File: Typecipt Programmimg/Selection1.ts

    function CheckEven(No : number): void
    {
        if((No % 2) == 0)
        {
            console.log("It is even number");
        }
        else 
        {
            console.log("It is odd number");
        }
    }

    var Value1 : number = 11;
    var Value2 : number = 10;

    CheckEven(Value1);
    CheckEven(Value2);